js/adapt-contrib-matching.js

```diff
@@ -98,26 +98,33 @@ define([
         },
 
         canSubmit: function() {
-
             var canSubmit = true;
 
-            $('.matching-select option:selected', this.el).each(_.bind(function(index, element) {
-
-                var $element = $(element);
-
-                if ($element.index() === 0) {
+            $('.matching-select option:selected', this.el).each(function(index, element) {
+                if ($(element).index() === 0) {
                     canSubmit = false;
```

Comment:
my bad, you can for `JQuery.each`; it's `_.each` and `array.forEach` where you can't